wikiHandler.js

```diff
@@ -78,6 +86,14 @@ function formatItemData(item_data) {
     return formatted_items;
 }
 
```

Comment:
I got lazy here and didn't fully explore your code.  formatted_items is doing a lot that just gets thrown away.  TODO.

Reply:
Uhh yeah, I'm trying to figure out what is changing but my brain hurts a little. Don't see anything missing though in the test server so maybe it's ok? Not sure what would be missing...

Reply:
Nothing is missing- it's actually the opposite: formatItemData also returns some values that are not being used.  Minor performance issue if we want to fix in the future.  I was thinking of including a  "price check" command at a later date, so we can just address this then.